bin/land.js

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const childProcess = require('child_process');
+const https = require('https');
+const path = require('path');
+const util = require('util');
+const clipboardy = require('clipboardy');
+
+const args = process.argv.slice(2);
+const exec = util.promisify(childProcess.exec);
+const targetBranch = args[0] || 'master';
+const toolName = path.basename(process.argv[1]);
+const toolVersion = require('../package.json').version;
+
+const help = `\
+This is a Pull Request landing tool that will automatically pick up commits from
+the branch adds metadata to them and merge them into the specified branch.
+
+Usage: ${toolName} <target-branch> [OPTION]
+       ${toolName} --help
+       ${toolName} --version
+
+Options:
+  --remote-name   get name from remote repo
+  --rebase        start interactive rebase of the source branch
+  --autosquash    move commits that begin with squash!/fixup! during rebase
+  --cherry-pick   apply commits from source branch onto <target-branch>
+  --clipboardy    print 'Landed in (...commitsHash)'
+  --help          print this help message and exit
+  --version       print version and exit
+`;
+
+const runGit = async function(options) {
+  return new Promise((resolve, reject) => {
+    const git = childProcess.spawn('git', options, {
+      stdio: 'inherit',
+      windowsHide: true,
+    });
+    git.on('close', code => {
+      if (code !== 0) {
+        reject(new SyntaxError('git exit with exit code !== 0'));
+      }
+      resolve();
+    });
+  });
+};
+
+async function commitsHash(count) {
+  let git;
+  try {
+    git = await exec(`git log -${count} --pretty=format:%h`);
+  } catch (error) {
+    console.error(error);
+    process.exit(1);
```

Comment:
Nit: I'd prefer to handle these errors (here and in differCommits, currentBranch, getRepoName) at the calling site to avoid duplicate code (i.e. `console.error(error); process.exit(1)`).